coral/src/domain/environment/environment-api.ts

```diff
@@ -26,9 +26,7 @@ const getEnvironmentsForTopicRequest = async (): Promise<Environment[]> => {
 
 const getEnvironmentsForTopicRequestByTeam = (): Promise<Environment[]> => {
```

Comment:
As we are using a different API, should it also be reflected in the API function name?

Reply:
I am going to go ahead and remove the api call and replace with getEnvironmentsForTopicRequest which is using the correct API call and is already there and defined, I can delete the rest